client/my-sites/github-deployments/deployments/styles.scss

```diff
@@ -136,6 +136,7 @@
 	gap: 4px;
 	/* stylelint-disable-next-line scales/radii */
 	border-radius: 12px;
+	cursor: pointer;
```

Comment:
It's weird we need to add this to a button?

Reply:
I think because it's a "plain" button it remove all the default styles